src/components/modules/banners/contributions/ContributionsBanner.tsx

```diff
@@ -184,6 +186,12 @@ const ContributionsBanner: React.FC<ContributionsBannerProps> = ({
     );
 };
 
-const wrapped = contributionsBannerWrapper(ContributionsBanner);
+const validate = (props: unknown): props is BannerProps => {
+    const result = bannerSchema.safeParse(props);
+    return result.success;
+};
+
+const banner = contributionsBannerWrapper(ContributionsBanner);
+const wrapped = withParsedProps(banner, validate);
 
-export { wrapped as ContributionsBanner };
+export { wrapped as ContributionsBanner, banner as ContributionsBannerUnwrapped };
```

Comment:
Does `ContributionsBanner` cause problems with storybook?

Reply:
After the `withParsedProps` hoc the `ContributionsBanner` has prop type of `unknown` so you have to start getting some `@ts-ignore`s in there and it was getting a little messy

Reply:
Fair enough.
Small thing, but 'wrapped' is getting a bit overloaded here. Perhaps `ContributionsBannerNonValidated` or something